modules/ting_oembed/ting_oembed.info

```diff
@@ -7,3 +7,4 @@ version = 7.x-1.x
 ;files[] =
 dependencies[] = oembed
 dependencies[] = ting_oembed_features
+dependencies[] = virtual_field
```

Comment:
You are absolutely right to add `virtual_field` as a dependency here, but then we can remove it from `ting_oembed_features.info` as well.